### DISCUSSION TITLE
How did you create the documentation? (capture animation for HTML display)

### DISCUSSION THREAD

**Discussion by THERESILIENCYGOVERNMENT** [spectreconsole/spectre.console#1337] [General]
**Is your feature request related to a problem? Please describe.**
how did you generate this page?

https://spectreconsole.net/live/status

**Comment by nils-a** [comment]
The whole website is generated using [Statiq](https://www.statiq.dev). All the sources for the website are in the [docs](https://github.com/spectreconsole/spectre.console/tree/0.47.0/docs) folder and specifically the status page is at https://github.com/spectreconsole/spectre.console/blob/0.47.0/docs/input/live/status.md
